Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean

```diff
@@ -1318,6 +1371,12 @@ lemma fromAffine_some [Nontrivial R] {X Y : R} (h : W'.toAffine.Nonsingular X Y)
     fromAffine (.some h) = ⟨(nonsingularLift_some ..).mpr h⟩ :=
   rfl
 
+lemma fromAffine_ne_zero [Nontrivial R] {X Y : R} (h : W'.toAffine.Nonsingular X Y) :
+    fromAffine (.some h) ≠ 0 := fun h0 ↦ by
+  erw [Point.ext_iff, Quotient.eq] at h0
+  obtain ⟨u, eq⟩ := h0
```

Comment:
```suggestion
  obtain ⟨u, eq⟩ := Quotient.eq.mp <| (Point.ext_iff ..).mp h0
```
Once we have the equivalence between affine and Jacobian coordinates, this follows from the fact that `.some` isn't `0`, but I guess you want this for non-trivial rings too?